GoogleForCreators/web-stories-wp
# Allow data removal on uninstall

## Summary

Adds an uninstall.php file that allows removing all Web Stories related data when deleting the plugin.

Right now, data removal is opt-in with a filter.

## Relevant Technical Choices



## To-do


## User-facing changes

None

## Testing Instructions

Turn on `web_stories_erase_data_on_uninstall` and try to delete the plugin.

---



See #1822
